src/lib.rs

```diff
@@ -168,6 +168,9 @@ mod types;
 mod utils;
 mod xor_name;
 
+/// Reexports `crust::Config`
+pub type BootstrapConfig = crust::Config;
```

Comment:
This is presented to the API of safe_core which presents it to frontend - Safe-core has not mention of crust in it and frontend is also agnostic to that term - so a more generic name was chosen to be more meaningful to them to what it is.